docs/blogs/kubecon_cfp.md

```diff
@@ -0,0 +1,45 @@
+# ElasticDL: Kubernetes-native Deep Learning System with Elastic Scheduling
+
+## Description
+
+In these years, Kubernetes becomes the fundamental infrastructure of machine
+learning. Kubeflow project provides some operators to run distributed machine
+learning jobs on Kubernetes. It uses gang scheduling policy. For example, a job
+requires N pod, it will keep pending if there are N-1 or less available pods. And
+during execution, the job will crash if any one of these N instances crashes. As
+a result, the job pending time is long and the resouce utilization is insufficient.
+
+ElasticDL implements elastic scheduling policy. A job doesn't need wait for all
```

Comment:
ElasticDL, a distributed deep learning framework, changes the paradox by using the statistical properties of distributed learning theory and making jobs tolerable to varying numbers of workers.  Take it as the basis, ElasticDL realizes elastic scheduling by introducing a master pod per job, to replace the Kubernetes operator per cluster.  The master takes three roles.

1. The master dynamically partitions the training data so to decouple the number of partitions and that of the workers.  
1. The master also works with Kubernetes to watch the cluster utilization and a good chance to restart failed workers.  Before the chance, the master always leveraging living workers.
1. The master starts and monitors parameter servers when training large models using the asynchronous SGD algorithm, and cooperate workers to implement a Kubernetes-native fault-tolerable AllReduce operation for the synchronous SGD counterpart.

Reply:
Moved the technical details of master pod into the benefit part.